apps/api-gateway/src/public-data/public-data.controller.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { NotFoundException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { PostCategory } from 'apps/public-data/src/public-data/entities/post-category.entity';
+import { firstValueFrom, of } from 'rxjs';
+
+import { PublicDataController } from './public-data.controller';
+import { PublicDataService } from './public-data.service';
+
+describe('PublicDataController', () => {
+  let controller: PublicDataController;
+  let service: PublicDataService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [PublicDataController],
+      providers: [
+        {
+          provide: PublicDataService,
+          useValue: {
+            getPostData: jest.fn(),
+            getOnePostData: jest.fn(),
+          },
+        },
+      ],
+    }).compile();
+    controller = module.get<PublicDataController>(PublicDataController);
+    service = module.get<PublicDataService>(PublicDataService);
+  });
+
+  it('api-gateway/public-data controller should be defined', () => {
```

Comment:
과도한 정보가 제공되는 것 같다는 생각도 드네요.
어떤 컨트롤러에 대한 테스트 코드인지는 파일명으로 충분히 표현이 되기 때문에 요런 느낌도 괜찮을 것 같아요.
```suggestion
  it('controller should be defined', () => {
```